test/jdk/java/nio/file/WatchService/SensitivityModifier.java

```diff
@@ -108,7 +120,10 @@ static void doTest(Path top) throws Exception {
                 if (eventReceived) {
                     System.out.println("Event OK");
                 } else {
-                    throw new RuntimeException("No ENTRY_MODIFY event received for " + file);
+                    Path parent = file.getParent();
+                    String msg = String.format("No ENTRY_MODIFY event received for %s (dir: %s, sensitivity: %d)",
+                            file, parent, pathToTime.get(parent));
+                    throw new RuntimeException(msg);
```

Comment:
Not clear to me that this helps. Have you looked at changing the 1s at L104. I assume the issue is that the test needs the idle (no events) time increased to deal with cases where the watcher is slow to queue events.

Reply:
That's a better idea but I think this change should go in also. I have not in any case been able to reproduce it. There is no record of on which specific machines it was sighted.

Reply:
Okay.